AEPEdgeConsent.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "AEPEdgeConsent"
-  s.version          = "1.0.0"
+  s.version          = "1.0.1"
   s.summary          = "Experience Platform Consent Collection extension for Adobe Experience Platform Mobile SDK. Written and maintained by Adobe."
 
   s.description      = <<-DESC
```

Comment:
Let's fix the homepage in podspec 
`s.homepage = "https://github.com/adobe/aepsdk-edgeconsent-ios.git"` instead of
`s.homepage = "https://github.com/adobe/aepsdk-consentedge-ios.git"`

Reply:
Same for `s.source = "https://github.com/adobe/aepsdk-edgeconsent-ios.git"`

Reply:
Thanks for the reminder. I've opened PR #55 for this so it will go into the dev branch. Once that's merged in I will merge this PR.